src/components/shared/SvgSprite.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+
+interface TProps {
```

Comment:
why sometimes naming TProps or Props

Reply:
TProps I usually use if props are described by type, but here this entry is incorrect, because here the interface